### PR TITLE
executor: refine `EXPLAIN ANALYZE` info for `INSERT IGNORE` statement

### LINKED ISSUE #38769: Refine EXPLAIN ANALYZE information for INSERT IGNORE statement.

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

Issue Number: close #38769

example:

prepare:

```sql
set @@global.tidb_enable_foreign_key=1;
set @@foreign_key_checks=1;
create table t1 (id int key);
create table t2 (id int key);
create table t3 (id int key, id1 int, id2 int, constraint fk_id1 foreign key (id1) references t1 (id) on delete cascade, constraint fk_id2 foreign key (id2) references t2 (id) on delete cascade);
insert into t1 values (1), (2);
insert into t2 values (1);
```

Now you can see `fk_check: 54.8µs, fk_num: 3` information in `Isert` execution info.
```sql
mysql>explain analyze insert ignore into t3 values (1, 1, 1), (2, 1, 1), (3, 2, 1), (4, 1, 1), (5, 2, 1), (6, 2, 1);
+----------+---------+---------+------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------+---------+------+
| id       | estRows | actRows | task | access object | execution info                                                                                                                                                                                   | operator info | memory  | disk |
+----------+---------+---------+------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------+---------+------+
| Insert_1 | N/A     | 0       | root |               | time:113µs, loops:1, prepare: 12.6µs, check_insert: {total_time: 100.4µs, mem_insert_time: 90.9µs, prefetch: 9.58µs, fk_check: 54.8µs, fk_num: 3, rpc:{BatchGet:{num_rpc:3, total_time:19.8µs}}} | N/A           | 1.90 KB | N/A  |
+----------+---------+---------+------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------+---------+------+
```

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: executor/insert_common.go [hunk 1]
```diff
@@ -1146,6 +1150,7 @@ func (e *InsertValues) batchCheckAndInsert(ctx context.Context, rows [][]types.D
 	}
 
 	if e.stats != nil {
+		e.stats.FKCheckTime += prefetchStart.Sub(start)
```
Comment (lcwangchao) [inline executor/insert_common.go:1153]: FKCheckTime includes the txn active time, may be not so accurate to indicate "fk check time"

**Review by lcwangchao — APPROVED** [review]

File: executor/insert_common.go (hunk 1 above)
Comment (crazycs520) [reply executor/insert_common.go:1153]: The txn is already been activated when the executor open, so it won't cost many time here.

**Comment by hawkingrei** [comment]
@crazycs520 

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x3e47276]

goroutine 113000 [running]:
testing.tRunner.func1.2({0x481cfc0, 0x748b9e0})
	GOROOT/src/testing/testing.go:1396 +0x24e
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1399 +0x39f
panic({0x481cfc0, 0x748b9e0})
	GOROOT/src/runtime/panic.go:884 +0x212
github.com/pingcap/tidb/executor.(*InsertRuntimeStat).String(0xc0043a8f28)
	executor/insert_common.go:1344 +0x4d6
github.com/pingcap/tidb/executor_test.TestInsertRuntimeStat(0x0?)
	executor/insert_test.go:1544 +0xba
testing.tRunner(0xc03659cb60, 0x4f55db8)
	GOROOT/src/testing/testing.go:1446 +0x10b
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1493 +0x35f
```

**Pull request merged**